webpack.config.babel.js

```diff
@@ -11,6 +11,7 @@ const PATH_DEMO = join(__dirname, 'demo')
 const PATH_SRC = join(__dirname, 'src')
 const PATH_INDEX = join(__dirname, 'index.html')
 const PATH_TESTS = join(__dirname, 'test', 'specs')
+const VIDEOJS_VAST = join(__dirname, 'node_modules', 'maestro-videojs-vast', 'dist')
```

Comment:
what's this doing?

Reply:
shouldn't all this just come in from the dependency?

Reply:
narrows down the files to include in the parsers